includes/admin/reporting/class-sales-logs-list-table.php

```diff
@@ -52,6 +52,18 @@ public function __construct() {
 	}
 
 	/**
+	 * Gets the name of the default primary column.
+	 *
+	 * @since 2.5
+	 * @access protected
+	 *
+	 * @return string Name of the default primary column.
+	 */
+	protected function get_default_primary_column_name() {
+		return 'ID';
```

Comment:
^ my comment was meant for the payments table file. But same thing applies here.